gtsfm/visualization/view_scene_olsson.py

```diff
@@ -0,0 +1,81 @@
+"""
+Script to render a GTSFM scene using Open3d.
```

Comment:
sorry I did not see this file earlier. how is it different from existing visualizers?

Reply:
could you add more documentation on how this is different?

Reply:
Sure, I just re-wrote the docstring at the top of the file.